### PR TITLE
Current Month Task Analysis

### PR DESCRIPTION
**Pull request by rishabh3**
#### Short description of what this resolves:
Analysis of Current Months Task as in what percentage is completed and what is not , How frequently task has been added etc
#### Changes proposed in this pull request:

-Added code segment to analyse all the files and separate current  month and year files for the task.
-Added code segment to compute the percentages and frequency as in total task in a month per day

**Fixes**: #
Fixed index out of bound error in the completing task command when user gives task number greater than the number of task.

### CODE REVIEW

**Comment by manparvesh** [comment]
Hi,
Thanks for the PR  
Almost everything seems fine. Can you please describe what do you mean by the frequency of adding tasks? I see what you are doing, but it is not very understandable by someone unless they see the code.  
It is better to append the unit of frequency you are using here.

**Comment by manparvesh** [comment]
For editing the PR you just need to push your changes to your forked branch and it will all be shown here.
Just let me know when you are done with it

**Comment by rishabh3** [comment]
@manpravesh i have made the changes and committed it

File: modules/learn.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def learn():
 
 # getting words
 words = {}
-with open('resources/vocab-words.txt') as fp:
+with open('/home/rishabh/Developer/python/dude/resources/vocab-words.txt') as fp:
```
Comment (manparvesh) [inline modules/learn.py]: This won't work for others' systems. Please remove this part. 
We will fix this in a different PR
For now, focus on the Issue you are fixing

File: modules/learn.py [hunk 2]
```diff
@@ -464,7 +464,7 @@ def define(word):
             if word in words:
                 chalk.blue('This word already exists in the vocabulary set, so you can practice it while using that')
             else:
-                with open('resources/vocab-words.txt', 'a') as fp:
+                with open('/home/rishabh/Developer/python/dude/resources/vocab-words.txt', 'a') as fp:
```
Comment (manparvesh) [inline modules/learn.py]: same here

**Comment by rishabh3** [comment]
Yeah i forgot to remove that

**Comment by rishabh3** [comment]
I have made those changes

**Pull request merged**